### PR TITLE
Add maint repos in console after installation than during installation

### PR DESCRIPTION
**Pull request by dzedro**
Independent of distro version and installation graphical mode,
faster and smaller chance to fail e.g. because of needle or race condition
I noticed pretty much [QAM test failures](https://openqa.suse.de/tests/overview?version=12-SP3&groupid=108&build=20181018-1&distri=sle) on add_update_test_repo

- Related ticket: https://progress.opensuse.org/issues/41921
- Verification run: http://10.100.12.155/tests/8223

### CODE REVIEW

File: tests/console/add_update_test_repo.pm [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+use strict;
+use base "y2logsstep";
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
```
Comment (kravciak) [inline tests/console/add_update_test_repo.pm]: I think testapi `get_var_array` does the some

Comment (dzedro) [reply tests/console/add_update_test_repo.pm]: thank you, changed

File: tests/console/add_update_test_repo.pm [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+use strict;
+use base "y2logsstep";
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+
+    my @repos = get_var_array('MAINT_TEST_REPO');
+    while (defined(my $maintrepo = shift @repos)) {
```
Comment (kravciak) [inline tests/console/add_update_test_repo.pm]: Same as `foreach my $maintrepo in (@repos)`, right? LGTM

**Comment by coolo** [comment]
smaller chance to fail == missing bugs.

The whole point of testing *installation* on unreleased incidents is because that's what customers will do once it's released: install with these packages in repo.

If you always install GA, you can just as well skip installation and use a silver image to install updates on. It's basically a different beast.

**Comment by dzedro** [comment]
But customer will not install new system, he will do basically the same with `zypper up`.
There are another tests that are doing installations, the problem here is that when there are e.g. 100 updates and test will fail because of needle or some race condition that sux :disappointed: 
I don't push this, it was just idea before the autoyast way is done

**Comment by coolo** [comment]
'But customer will not install new system'? How do you think servers end up in server rooms?

**Comment by dzedro** [comment]
I did not say never, I'm pretty sure customers don;t buy new hardware for every update
@coolo got you point

**Pull request closed without merging**